content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,101 @@
+use. TBD: LINK
+ 
+- **Tracing Improvements**. There have been many improvements in our overall tracing story:
+
+    - We now use 128 bit trace ids
+    - There are some enhanced deployment options for Jaeger and Zipkin TBD: LINKS
+    - We now support sending trace data to LightStep
+    - Enabled full disablement of tracing for Mixer-backed services TBD: REPHRASE
+    - Added policy decision-aware tracing. TBD: LINK
+
+- **Default TCP Metrics**. There are now default metrics for tracking TCP connections.
+
+## Configuration management
+
+- **Galley**. Galley is now used as the primary config ingestion and distribution mechanism within Istio. It provides
+a robust model to validate, transform, and distribute configuration state to Istio components, insulating the components
+from Kubernetes details. Galley uses the [Mesh Configuration Protocol (MCP)](https://github.com/istio/api/tree/{{< source_branch_name >}}/mcp) to interact with components. TBD: LINK TO MCP
 
-## `istioctl`
+- **Monitoring Port**. Galley's default monitoring port was changed from 9093 to 15014.
 
-- Deprecated `istioctl create`, `istioctl replace`, `istioctl get`, and `istioctl delete`. Use `kubectl` instead (see <https://kubernetes.io/docs/tasks/tools/install-kubectl>). The next release (1.2) removes the deprecated commands.
-- Deprecated `istioctl gen-deploy`. Use a [`helm template`](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template) instead. The next release (1.2) removes this command.
+## `istioctl` and `kubectl`
 
-- Added [`istioctl validate`](/docs/reference/commands/istioctl/#istioctl-validate) for offline validation of Istio Kubernetes resources. The intent is to replace the existing use of the deprecated `istioctl create` command.
+- **Validate Command**. Added the [`istioctl validate`](/docs/reference/commands/istioctl/#istioctl-validate) command for offline validation of Istio Kubernetes resources. This replaces
 
-- Added [`istioctl experimental verify-install`](/docs/reference/commands/istioctl/#istioctl-experimental-verify-install). This experimental command verifies the installation status of Istio given a specified install YAML file.
+- **Verify-Install Command**. Added the [`istioctl experimental verify-install`](/docs/reference/commands/istioctl/#istioctl-experimental-verify-install) command used to verify the status of an 
+Istio installation given a specified installation YAML file.
 
-## Configuration
+- **Deprecated Commands**. The `istioctl create`, `istioctl replace`, `istioctl get`, and `istioctl delete` commands are being deprecated. Use [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) instead.
+The `istioctl gen-deploy` command is also being deprecated. Use a [`helm template`](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template) instead.
+This command will be removed completely in the 1.2 release.
```

Comment:
Which command? There are at least 5 different commands mentioned on this bullet.

Reply:
Changed to plural.